builder/builder-next/exporter/exporter.go

```diff
@@ -0,0 +1,3 @@
+package exporter
+
+const Moby = "moby"
```

Comment:
Same as for the other strings I mentioned above; we need to figure out what the canonical source of truth is for this string; I see there's uses of `"moby"` in the BuildKit and buildx repository as well (I didn't dig deep if those are for the same purpose);

```bash
git grep '"moby"'
2022
client/client_test.go:                  Type: "moby",
client/client_test.go:                          Type: "moby",
client/client_test.go:                  Type: "moby",
frontend/dockerfile/dockerfile_buildinfo_test.go:                       Type: "moby",
frontend/dockerfile/dockerfile_buildinfo_test.go:                       Type: "moby",
frontend/dockerfile/dockerfile_buildinfo_test.go:                       Type: "moby",
frontend/dockerfile/dockerfile_buildinfo_test.go:                       Type: "moby",
frontend/dockerfile/dockerfile_buildinfo_test.go:                       Type: "moby",
```

```bash
git grep '"moby"'
build/build.go:                 opt.Exports[i].Type = "moby"
build/build.go:                                 if e.Type == "moby" {
build/build.go:                                                         if e.Type == "moby" && e.Attrs["push"] != "" {
```

Reply:
Buildkit's one are only tests.

The buildx use of `"moby"` is for the same thing, the exporter name. Looking at all the usages of `.Type` in buildx it looks like they are using strings for everything everywhere. 

The exporter type `moby` is defined by moby so it should live in our code.

Reply:
>  Looking at all the usages of .Type in buildx it looks like they are using strings for everything everywhere.

Yeah, I noticed a lot of those; started to tackle some of them in https://github.com/moby/buildkit/pull/3199 some time ago, but still trying to get it reviewed.

> The exporter type moby is defined by moby so it should live in our code.

Makes sense 👍